testStrategies/commonToAllTestStrategies.go

```diff
@@ -407,7 +407,8 @@ func getArrayNameAndIndex(testRunGlobals map[string]interface{}, propPathPart st
 	propPathPart = strings.Replace(propPathPart, "]", "", 1)
 	propertyNameParts := strings.Split(propPathPart, "::")
 
```

Comment:
Moved the comment to suggest that the line of code is handling the error as needed. This is already the behavior we want and would just set index = 0 on exception anyway.